### PR TITLE
Remove prepare script and revert PR #20

### PR DESCRIPTION
**Pull request by hbcarlos**
Remove prepare script and revert PR #20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +83,9 @@
       "before-build-npm": [
         "python -m pip install jupyterlab~=3.0",
         "jlpm"
+      ],
+      "before-build-python": [
+        "jlpm clean"
```
Comment (jtpio) [inline package.json:88]: Let's check if this forces a clean build.

**Comment by jtpio** [comment]
Looking better after forcing the `clean:all` before the `build-python` step:

image

We should also double check if it builds in dev mode because of the first `python -m pip install .` in `prep-git`: 

image

**Comment by hbcarlos** [comment]
Thanks @jtpio!!!

**Comment by hbcarlos** [comment]
> We should also double check if it builds in dev mode because of the first python -m pip install . in prep-git:

Yes, it is probably coming from there. Do you know if there is a reason to use the editable mode there?
https://github.com/jupyter-server/jupyter_releaser/blob/ca540f3441045db34d1b433c8a0714a9f97648ba/jupyter_releaser/lib.py#L565-L568

**Comment by jtpio** [comment]
> Do you know if there is a reason to use the editable mode there?

Would have to investigate further to confirm, but a first guess could be so that it picks up later changes such as the version bump of the Python package.

**Comment by jtpio** [comment]
> Would have to investigate further to confirm, but a first guess could be so that it picks up later changes such as the version bump of the Python package.

Opened https://github.com/jupyterlab/extension-cookiecutter-ts/issues/199 as follow-up.

**Pull request merged**